source/common/config/new_grpc_mux_impl.h

```diff
@@ -41,7 +41,13 @@ class NewGrpcMuxImpl
                            SubscriptionCallbacks& callbacks,
                            OpaqueResourceDecoder& resource_decoder,
                            const bool use_namespace_matching = false) override;
-
+  /*
```

Comment:
?

Reply:
Legacy code from the original implementation. Will clean it up.